WowPacketParser/SQL/Builder.cs

```diff
@@ -68,6 +68,8 @@ public static void DumpFile(string prefix, string fileName, string header, List<
             var startTime = DateTime.Now;
             using (var store = new SQLFile(fileName))
             {
+                store.WriteData(UnitMisc.CreatureEquip(units)); // ensure this is run before spawns
```

Comment:
i kinda don't like this, but this is required to also allow equipment which has not been added to database yet but is in the same sniff